packages/website/pages/components/[slug].tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import ErrorPage from 'next/error';
+import { remark } from 'remark';
+import html from 'remark-html';
+import { css } from 'emotion';
+
+import * as f36Components from '@contentful/f36-components';
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
+
+import { geltAllMDX, getPageBySlug } from '../../utils/content';
+
+const styles = {
+  root: css({
+    minHeight: '100vh',
+    padding: '0 0.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  }),
+};
+
+export default function ComponentPage(props: {
+  data: any;
```

Comment:
I didn't type this because it was an experiment, but now I think we should do it

Reply:
I'm going to remove it in the follow up PR

Reply:
hahaha I think you went too hard on the copy and paste, I don't want this to be removed

I'm suggesting this to be typed. If you don't want to type it then put an `unknown` I guess

Reply:
it will be replaced in the next PR :) I already have this next branch locally